lib/Runtime/Library/WasmLibrary.cpp

```diff
@@ -0,0 +1,74 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+//-------------------------------------------------------------------------------------------------------
+#include "RuntimeLibraryPch.h"
+
+#ifdef ENABLE_WASM
+namespace Js
```

Comment:
I don't think I can, the macro
```
BUILTIN(Wasm::WasmLibrary, instantiateModule, instantiateModule, FunctionInfo::ErrorOnNew)
```
doesn't accept the `Wasm` namespace because it uses this to make an enum and without the namespace it doesn't find the class.
I could do a `using namespace Wasm;` where the macro is used, but I doubt this is a very good idea